pkg/groups/gsuite/grouper.go

```diff
@@ -110,16 +112,18 @@ func (g *Grouper) Members(group string) ([]rbacv1.Subject, error) {
 	client, err := g.service(ctx)
 	if err != nil {
 		return nil, errors.Wrapf(groups.ErrUnknown,
-		"unable to determine group members, an error occurred creating gsuite client: %v",
-		err)
+			"unable to determine group members, an error occurred creating gsuite client: %v",
+			err)
 	}
+	metrics.RBACSyncGsuiteClientCreationStatus.Inc()
```

Comment:
This is not incremented in the error case which I think is more useful.  We can add labels with success or failure (or status codes).  I think users would want to alert on failures of client creation and would care less about the number of successes.

Reply:
sounds good

Reply:
updated, thanks for pointing out.